src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -2048,18 +2047,24 @@ private void ParseExpectedErrorConfigurations()
 
             foreach (var errorClass in _localConfiguration.errorCollector.expectedMessages)
             {
-                var messages = errorClass.message;
-                if (messages != null)
+                if (errorClass.message != null)
```

Comment:
what happens if errorClass.message is empty?

Reply:
I believe it will set any errors from that class to be expected.